pkg/assembler/backends/testing/src.go

```diff
@@ -17,120 +17,285 @@ package testing
 
 import (
 	"context"
-	"strings"
+	"fmt"
+	"log"
+	"strconv"
 
 	"github.com/guacsec/guac/pkg/assembler/graphql/model"
 	"github.com/vektah/gqlparser/v2/gqlerror"
 )
 
 func registerAllSources(client *demoClient) {
-	// with tag
-	client.registerSource("git", "github", "github.com/guacsec/guac", "tag=v0.0.1")
-	// with commit
-	client.registerSource("git", "github", "github.com/guacsec/guac", "commit=fcba958b73e27cad8b5c8655d46439984d27853b")
-	// with no tag or commit
-	client.registerSource("git", "github", "github.com/guacsec/guac", "")
-	// gitlab namespace
-	client.registerSource("git", "gitlab", "github.com/guacsec/guacdata", "tag=v0.0.1")
-	// differnt type
-	client.registerSource("svn", "gitlab", "github.com/guacsec/guac", "")
-	// "git", "github", "https://github.com/django/django", "tag=1.11.1"
-	client.registerSource("git", "github", "https://github.com/django/django", "tag=1.11.1")
-	// "git", "github", "https://github.com/vapor-ware/kubetest", "tag=0.9.5"
-	client.registerSource("git", "github", "https://github.com/vapor-ware/kubetest", "tag=0.9.5")
+	ctx := context.Background()
```

Comment:
Same minor nit here about deleting this

Reply:
I am going to put a todo here and instead of deleting, add it to the unit tests